src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -283,6 +283,100 @@ namespace Sapphire::Network::Packets::Server
     char unk2[0x4]; // This has probs something to do with the support desk (inquiry id?)
   };
 
+  struct FFFXIVIpcMarketBoardItemListingCount : FFXIVIpcBasePacket< MarketBoardItemListingCount >
+  {
+    uint32_t itemCatalogId;
+    uint32_t unknown1; // does some shit if nonzero
+    uint16_t requestId;
+    uint16_t quantity; // high/low u8s read separately?
+    uint32_t unknown3;
+  };
+
+  struct FFXIVIpcMarketBoardItemListing : FFXIVIpcBasePacket< MarketBoardItemListing >
+  {
+    struct itemListing // 152 bytes each
+    {
+      uint32_t unknown; // Changes if multiple packets are sent for a single item
+      uint8_t padding;
+      uint16_t unknown1;
+      uint8_t padding1;
+      struct unknown2
+      {
+        uint32_t unknown3;
+        uint16_t unknown4;
+        uint16_t unknown5;
+      } unknown6[3];
+      uint32_t pricePerUnit;
+      uint32_t unknown7;
+      uint32_t itemQuantity;
+      uint32_t itemID;
+      uint16_t unknown8;
+      uint16_t unknown9;
+      uint32_t unknown10;
+      uint16_t unknown11; // Always 30000
```

Comment:
durability, 30000 is 100%, 60000 is 200%

market might reset durability to 100% for buyers?